src/components/project-pages/EroticStories.tsx

```diff
@@ -1,33 +1,128 @@
 import React from "react";
 import styled from "styled-components";
-import Timer from "../Timer";
-import { FlexboxProps, flexbox, layout, LayoutProps } from "styled-system";
-import statue from "../assets/project-page/statue.png";
+import { FlexboxProps, flexbox, layout, LayoutProps, grid, GridProps, space, SpaceProps, typography, TypographyProps } from "styled-system";
+import EroticStoriesPreview from "./EroticStoriesPreview";
+import alasImg from "./../assets/erotic-stories/alas-bateando.jpg";
```

Comment:
this is unused?

Reply:
We're going to use that image I'm just waiting on a replacement from Aitor because that's a broken file

Reply:
Also why I've used the same image twice in the grid

Reply:
I can remove it for now as its not being used